ssl/tls13_both.cc

```diff
@@ -384,6 +402,83 @@ bool tls13_process_finished(SSL_HANDSHAKE *hs, const SSLMessage &msg,
   return true;
 }
 
+bool tls13_add_certificate_pha(SSL *ssl) {
+  assert(!ssl->server);
```

Comment:
Are the functions in `tls13_both.cc` expected to be for "both" client and server?  Should this function be in `tls13_client.cc`?